pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -190,10 +189,8 @@ def _init_deterministic(self, deterministic: bool) -> None:
         self.deterministic = deterministic
         if _TORCH_GREATER_EQUAL_1_8:
             torch.use_deterministic_algorithms(deterministic)
-        elif _TORCH_GREATER_EQUAL_1_7:
+        else:
             torch.set_deterministic(deterministic)
-        else:  # the minimum version Lightning supports is PyTorch 1.6
-            torch._set_deterministic(deterministic)
```

Comment:
This would break Lightning with 1.6 right ?

Reply:
Support was removed in #10367